IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EndPointId_3ctuSusCj2_w79HgKqMiDN" Name="EndPointId" UniqueId="e6df8738-d8cb-42fb-a1c9-46a5345ac357" PrimaryDataType="IfcIdentifier" />
```

Comment:
Is this a property? Or a relationship such as port connectivity.  It shouldn't be in a type driven override pset as I understand it.

Reply:
Backlog